iron-doc-property-styles.html

```diff
@@ -9,6 +9,25 @@
 -->
 
 <link rel="import" href="../paper-styles/typography.html">
+<link rel="import" href="../paper-styles/color.html">
+
+<!--
+### Styling
+
+The following custom properties and mixins are available for styling:
+
+Custom property | Description | Default
+----------------|-------------|----------
+`--iron-doc-viewer-deeplink-color` | Text color of clickable properties and events names | `currentcolor`
```

Comment:
Oof, sorry to be annoying, but do you think these should be `--iron-doc-viewer-property-xxx` and the other ones just `--iron-doc-viewer-xx` since they affect different components?